package.json

```diff
@@ -13,6 +13,7 @@
     "stylelint": "^16.0.2"
   },
   "dependencies": {
+    "common-tags": "^1.8.2",
```

Comment:
Sorry that I missed this in the previous review round, but this should be in `devDependencies`, not in `dependencies`.